### PR TITLE
:book instruction completed, description, screens added

### PR DESCRIPTION
**Pull request by Mirgalimova**

### CODE REVIEW

File: odoo_backup_sh/doc/index.rst [hunk 1]
```diff
@@ -29,46 +29,45 @@ Configuration
...
-
-Below are the forms for managing and controlling backups of your databases.
-In addition to auto backup, you can make new backup manually at any time.
-Backups taken by hand are not involved in auto rotation conditions.
-
-.. todo:: Add the section when interfaces are ready.
-
-{Instruction for daily usage. It should describe how to check that module works. What shall user do and what would user get.}
+* Open the menu ``[[ Backups ]] >> Dashboard``
+* Login into your Odoo account
```
Comment (yelizariev) [inline odoo_backup_sh/doc/index.rst]: It's not clear what is "Odoo account". Say "odoo.com" explicitly

File: odoo_backup_sh/doc/index.rst [hunk 2]
```diff
@@ -29,46 +29,45 @@ Configuration
...
+* Click on `Restore via Odoo-backup.sh` button
```
Comment (yelizariev) [inline odoo_backup_sh/doc/index.rst]: Use special format for buttons https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#buttons

Comment (yelizariev) [reply odoo_backup_sh/doc/index.rst]: Anyway, there is no such button at `` Backups >> Dashboard`` menu
дичь

File: odoo_backup_sh/doc/index.rst [hunk 3]
```diff
@@ -29,46 +29,45 @@ Configuration
...
 
 After all required fields will be filled, click on the ``[Save]`` button.
 
 All databases with their settings are available in the ``[[ Backups ]] >> Configuration >> Dashboard`` section.
+
 Each database specified in these settings will be presented in the dashboard.
 
 * `Activate Developer Mode <https://odoo-development.readthedocs.io/en/latest/odoo/usage/debug-mode.html>`__
 
 Usage
```
Comment (yelizariev) [inline odoo_backup_sh/doc/index.rst]: Usage section should also contain an insturction how to make manual backup, how to download backup from the cloud

File: odoo_backup_sh/__init__.py [hunk 4]
```diff
@@ -2,3 +2,15 @@
...
+
+def uninstall_hook(cr, registry):
+    # During the unistall of the module, we have the error:
+    # 'update or delete on table "ir_act_server" violates foreign key constraint
+    # "ir_cron_ir_actions_server_id_fkey" on table "ir_cron"'. Due to this error the "odoo_backup_sh.config" table
+    # will not be deleted. Below the code allows you to delete all entries related to "ir_cron".
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    backup_crons = env['odoo_backup_sh.config.cron'].search([('model_name', '=', 'odoo_backup_sh.config')])
+    if backup_crons:
+        backup_crons.unlink()
```
Comment (yelizariev) [inline odoo_backup_sh/__init__.py]: Are you developer now?)

https://www.youtube.com/watch?v=lJ_CqsmVdbo

File: odoo_backup_sh_dropbox/README.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+========================
+ Odoo-backup.sh Dropbox
+========================
+
+The application allows managing database backups via Dropbox, without downloading and uploading an archive via a browser.
```
Comment (yelizariev) [inline odoo_backup_sh_dropbox/README.rst]: No, it does not. It allows making backups only. Not restoring it via /web/database/backups page
wrong

File: odoo_backup_sh_dropbox/doc/index.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+* Specify your folder path `/Backups`
+* Click on `Save` button
+
+Usage
+=====
+
+* Open the menu ``[[ Backups ]] >> Dashboard``
+* Proceed with ``[[ Backups ]] >> Backups`` where Dropbox databases are presented
+* Go to ``[[Backup]] >> Configuration >> Schedules and Rotations``
+* See ``[[Backup]] >> Notifications`` if new one is displayed
```
Comment (yelizariev) [inline odoo_backup_sh_dropbox/doc/index.rst]: Why? It's only for odoo-backups.sh service

File: odoo_backup_sh_dropbox/doc/index.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+RESULT: You can see the main Graph with the general statistics of all your backups are stored on a remote server. After this window a special form for managing and controlling backups of your databases specially *for Dropbox* is located.
+In addition to auto backup, you can make new backups manually at any time. Backups *taken by hand* are not involved in auto rotation conditions.
```
Comment (yelizariev) [inline odoo_backup_sh_dropbox/doc/index.rst]: Are you sure?

> Backups *taken by hand* are not involved in auto rotation conditions.

File: odoo_backup_sh_google_disk/README.rst [hunk 8]
```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+=============================
+ Odoo-backup.sh Google Drive
+=============================
+
+The application allows managing database backups via Google drive, without downloading and uploading an archive via a browser.
```
Comment (yelizariev) [inline odoo_backup_sh_google_disk/README.rst]: wrong

File: odoo_backup_sh_google_disk/doc/index.rst [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+* Open the menu ``[[ Backups ]] >> Dashboard``
+* Proceed with ``[[ Backups ]] >> Backups`` where Google drive databases are presented
+* ``[[ Backups ]] >> Backups``
+* Go to ``[[Backup]] >> Configuration >> Schedules and Rotations``
+* See ``[[Backup]] >> Notifications`` if new one is displayed
+
+
+RESULT: You can see the main Graph with the general statistics of all your backups are stored on a remote server. After this window a special form for managing and controlling backups of your databases specially *for Google drive* is located.
+In addition to auto backup, you can make new backups manually at any time. Backups *taken by hand* are not involved in auto rotation conditions.
```
Comment (yelizariev) [inline odoo_backup_sh_google_disk/doc/index.rst]: Are you sure?

File: odoo_backup_sh/static/description/index.html [hunk 10]
```diff
@@ -0,0 +1,191 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Odoo Backups </h2>
```
Comment (yelizariev) [inline odoo_backup_sh/static/description/index.html]: It must be the same as Name in the Manifest. 
I think to use "odoo-backup.sh" as a brand name for the service. Let me know if you have better idea

File: odoo_backup_sh/static/description/index.html [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+            <h3 class="oe_slogan"> Create, save, duplicate, delete, restore backups</h3>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <p class="oe_mt32">
+                Main feature: no need to download and upload a database archive via a browser.
```
Comment (yelizariev) [inline odoo_backup_sh/static/description/index.html]: Another feature is encrypting backup archives

File: odoo_backup_sh/static/description/index.html [hunk 12]
```diff
@@ -0,0 +1,191 @@
...
+                            -webkit-transform: rotate(-3deg);
+                            -o-transform: rotate(-3deg);
+                            -moz-transform: rotate(-3deg);
+                            -ms-transform: rotate(-3deg);">
+                    Tested on Odoo<br/>12.0 enterprise
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
```
Comment (yelizariev) [inline odoo_backup_sh/static/description/index.html]: This html needs some information about prices of the service an probably links to the extra modules (dropbox and google drive)

File: odoo_backup_sh_dropbox/static/description/index.html [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+            <h3 class="oe_slogan">Manage database backups via Dropbox</h3>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <p class="oe_mt32">
+                Main feature: no need to download and upload a database archive via a browser.
```
Comment (yelizariev) [inline odoo_backup_sh_dropbox/static/description/index.html]: wrong

File: odoo_backup_sh_google_disk/static/description/index.html [hunk 14]
```diff
@@ -0,0 +1,197 @@
...
+                - Go to <em>IAM& admin &rarr; Service account</em> <br/>
+                - Click on <b>Create</b> and add the <i>Private key</i> <br/>
+                - Choose the <i>Service account</i> or create a new one <br/>
+                - If you create a new one please define <em>Project &rarr; Owner</em> <br/>
+                - Define the name of Service account <br/>
+                - Start Odoo with <code>--load=web,odoo_backup_sh</code> or set the <code>server_wide_modules</code> option in the Odoo configuration file <br/>
+                - Define a key type as <i>JSON</i> <br/>
+                - Click <b>Create</b> (key is created automatically) <br/>
+                - In opened page *IAM& admin* click on <b>Service account</b> <br/>
+                - Remember the e-mail of your service account
```
Comment (yelizariev) [inline odoo_backup_sh_google_disk/static/description/index.html]: Looks too boring for Presentation of the module

File: odoo_backup_sh/__init__.py (hunk 4 above)
Comment (Mirgalimova) [reply odoo_backup_sh/__init__.py]: Не поняла прикол(

File: odoo_backup_sh/static/description/index.html (hunk 12 above)
Comment (Mirgalimova) [reply odoo_backup_sh/static/description/index.html]: links provided

File: odoo_backup_sh/__init__.py (hunk 4 above)
Comment (yelizariev) [reply odoo_backup_sh/__init__.py]: В этом пул-реквести ты изменяешь код. Этого быть не должно и обновление кода может быть только отдельным пул-реквестом

File: odoo_backup_sh/doc/index.rst [hunk 15]
```diff
@@ -29,46 +29,60 @@ Configuration
...
-In addition to auto backup, you can make new backup manually at any time.
-Backups taken by hand are not involved in auto rotation conditions.
+After all required fields will be filled, click on the ``[Save]`` button.
+
+All databases with their settings are available in the ``[[ Backups ]] >> Configuration >> Dashboard`` section.
+
+Each database specified in these settings will be presented in the dashboard.
 
-.. todo:: Add the section when interfaces are ready.
+* `Activate Developer Mode <https://odoo-development.readthedocs.io/en/latest/odoo/usage/debug-mode.html>`__
```
Comment (yelizariev) [inline odoo_backup_sh/doc/index.rst]: What for?

File: odoo_backup_sh/doc/index.rst [hunk 16]
```diff
@@ -29,46 +29,60 @@ Configuration
...
+* All other options mark as **Disabled**.
 
-Below are the forms for managing and controlling backups of your databases.
-In addition to auto backup, you can make new backup manually at any time.
-Backups taken by hand are not involved in auto rotation conditions.
+After all required fields will be filled, click on the ``[Save]`` button.
+
+All databases with their settings are available in the ``[[ Backups ]] >> Configuration >> Dashboard`` section.
+
+Each database specified in these settings will be presented in the dashboard.
```
Comment (yelizariev) [inline odoo_backup_sh/doc/index.rst]: Whole section is in bad formatting

image

File: odoo_backup_sh/doc/index.rst [hunk 17]
```diff
@@ -29,46 +29,60 @@ Configuration
...
-* Click ``[{Button Name}]``
-* RESULT: {what user gets, how the modules changes default behaviour}
+Backups:
+
+* Open the menu ``[[ Backups ]] >> Dashboard``
+* After redireсtion please login into your `Odoo account <https://www.odoo.com/web/login>`__
+* Set up a new configuration  [[ Backups ]] >> Dashboard >> Add Database
+* Choose the Database in the database field
+* Encrypt your backup if you need, but do not forget the password
+* Enter *Interval Number*, *Interval Unit*, *New execution Date*
```
Comment (yelizariev) [inline odoo_backup_sh/doc/index.rst]: This is supposed to be in in Configuration section. Confifuration is something you do ideally once. Usage is something you do regurarly

File: odoo_backup_sh/static/description/index.html (hunk 10 above)
Comment (yelizariev) [reply odoo_backup_sh/static/description/index.html]: It was ignored

File: odoo_backup_sh/doc/index.rst (hunk 15 above)
Comment (Mirgalimova) [reply odoo_backup_sh/doc/index.rst]: Надо будет спросить у Динара)

File: odoo_backup_sh/doc/index.rst (hunk 16 above)
Comment (Mirgalimova) [reply odoo_backup_sh/doc/index.rst]: Окей, сейчас переформатирую

File: odoo_backup_sh/doc/index.rst (hunk 17 above)
Comment (Mirgalimova) [reply odoo_backup_sh/doc/index.rst]: То есть оставить в Usage только restore?

File: odoo_backup_sh/__init__.py (hunk 4 above)
Comment (Mirgalimova) [reply odoo_backup_sh/__init__.py]: Это видимо было разрешение конфликта при мердже с Динаром

File: odoo_backup_sh/doc/index.rst (hunk 17 above)
Comment (yelizariev) [reply odoo_backup_sh/doc/index.rst]: Ручное создание бакапов тоже нужно для регулярного использования. Плюс просмотр графиков \ списка бакапов

File: odoo_backup_sh_google_disk/static/description/index.html (hunk 14 above)
Comment (Mirgalimova) [reply odoo_backup_sh_google_disk/static/description/index.html]: Убрала boring configuration part из презентации модуля

**Pull request merged**